src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxconverter.py

```diff
@@ -11,7 +10,10 @@
 
 
 class SpikeGLXConverterPipe(ConverterPipe):
-    """Primary conversion class for handling multiple SpikeGLX data streams."""
+    """
+    The simplest, easiest to use class for converting all SpikeGLX data in a folder. Primary conversion class for
+    handling multiple SpikeGLX data streams.
+    """
```

Comment:
Comments about a class should be attached to the class, not the module. Also this renders much more sensibly in the auto doc